package.json

```diff
@@ -16,7 +16,8 @@
     "react-dom": "^18.2.0",
     "react-icons": "^4.10.1",
     "react-intersection-observer": "^9.5.2",
-    "react-spring": "^9.7.2"
```

Comment:
where you have used react-spring and styled components?

Reply:
Mobile View
[Screen Recording - Made with RecordCast (1).webm](https://github.com/DevSoc-Asansol2023/DevSoc-Main/assets/113552756/d56a17de-dc20-4540-af6b-7029f0b56f96)

Reply:
I have used useEffect and usestate hooks to do it. Is there any problem

Reply:
No its ok
Actually it was showing multiple versions of react spring
my bad I am merging it